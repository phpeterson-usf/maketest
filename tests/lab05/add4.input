$(repo_dir)/add4_main
