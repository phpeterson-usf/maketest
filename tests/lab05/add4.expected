add4_c(1, 2, 3, 4) = 10
add4_s(1, 2, 3, 4) = 10
add4_c(-1, -2, 0, -3) = -6
add4_s(-1, -2, 0, -3) = -6
